### ISSUE TITLE
NVDA unable to identify any content on certain Chrome pages

### ISSUE THREAD

**Issue by ssawczyn** [nvaccess/nvda#7944]
When loading certain web pages in Chrome, NVDA reads the page title, but none of the page content.  This holds true regardless of whether focus mode is on or off.  In addition, I'm unable to find any content using the object navigator.

Sample URL:  https://fafsa.ed.gov/

Alternative URL:  https://www.weightwatchers.com/us/checkout/?cid=eml_US_122617_Online_Lose10_Extra15_122617_Lapsed_Online_Winback_NACO_ExtraSavings&ct=hero&j=761152&sfmc_sub=30954105&l=352_HTML&u=8531653&mid=7003004&jb=210#/plan

Initially, I believed this was an issue with Chrome, so I opened a Chromium ticket here:  https://bugs.chromium.org/p/chromium/issues/detail?id=797695&can=2&start=0&num=100&q=&colspec=ID%20Pri%20M%20Stars%20ReleaseBlock%20Component%20Status%20Owner%20Summary%20OS%20Modified&groupby=&sort=

A few things to note:
* Pages that exhibit this behavior with Chrome and NVDA do not exhibit the same behavior in Firefox.
Behavior exists both in latest release version of Chrome and with Canary.
* I've not tested this, but according to a comment on the Chromium ticket I filed, this behavior does not exist with JAWS18 and Canary. Knowing that this works with another screen reader is what prompted me to file this bug here.

I have not been able to determine any sort of pattern with pages that cause this issue.  As I find others, I'll gladly add the URLs via comment here.

**Comment by Brian1Gaff** [comment]
As an aside, although both work in firefox 52, there is some  garbeled stuff 
about latin  said at the start which never seems to be there on other pages 
I access.
 Brian

**Comment by PratikP1** [comment]
It looks like the issue I reported in #7271 is back. I can replicate this.

**Comment by feerrenrut** [comment]
I can't reproduce this with chrome Version  `64.0.3282.119` or `63.0.3239.132` or with chrome canary version  `66.0.3334.0`

**Comment by PratikP1** [comment]
@feerrenrut This is strange. I'm able to replicate this behavior with the same version of Chrome Canary that you tried. Both of the links that @ssawczyn provided as test cases result in no apparent content loading on my machine. 
* Chrome Version 66.0.3334.0 (Official Build) canary (64-bit)
* NVDA version next-14811,143bd0f4
* Windows Insider 1709 built 17083.100.

**Comment by fernando-jose-silva** [comment]
I can reproduce this problem.
Chrome Version 64.0.3282.119 (Official version) 64-bit in português brazil
windows 7 64bits portuguese Brazil.
nvda next 14811 portuguese Brazil.

**Comment by fernando-jose-silva** [comment]
The call I opened from issue # 7928 I believe may have something there to do with this, the symptoms are the same.

**Comment by feerrenrut** [comment]
Could you add details about how you opened those pages, and how you interacted with them after the page loaded?

**Comment by fernando-jose-silva** [comment]
In the called pressi press on the link, the chrome opens the new page, and soon after the page was opened I pressed the arrows below, and the nvda repeats only the title of the page.

**Comment by ssawczyn** [comment]
I did the same as the others. I tried pasting the URL, and I also tried
activating the URL from another page just to  see if I would get different
results.  I also pasted the URL in the Windows run dialog (Chrome is my
default browser), same results regardless.

**Comment by Adriani90** [comment]
I cannot reproduce this issue in NVDA 2019.1.1 and Chrome 74. @ssawczyn, @fernando-jose-silva, @PratikP1 did you try to run the com registration fixing tool in nvda menu / tools? Are you still seeing this issue?

**Comment by fernando-jose-silva** [comment]
I have now been able to read the pages by opening the 2 links and navigating with the arrows below.
all updated
nvda 17110
windows 10 18362.
Thank you.

**Comment by Adriani90** [comment]
Closing as works for me.
